raiden-ts/docs-source/private-chain.md

```diff
@@ -1,68 +1,86 @@
 # Using the Light Client in different network setups
 
 ## Table of Contents
+
 - [Preface](#preface)
 - [Setting up the Light Client on a private network](#setting-up-the-light-client-on-a-private-network)
-  * [Identifying the correct smart contract version](#identifying-the-correct-smart-contract-version)
-  * [Locate the smart contract version](#locate-the-smart-contract-version)
+  - [Identifying the correct smart contract version](#identifying-the-correct-smart-contract-version)
+  - [Locate the smart contract version](#locate-the-smart-contract-version)
 - [Setting up the Development Environment](#setting-up-the-development-environment)
-  * [Locating the Smart Contracts](#locating-the-smart-contracts)
-  * [Starting the SDK](#starting-the-sdk)
-  * [Changing the dApp to work on the private setup](#changing-the-dapp-to-work-on-the-private-setup)
+  - [Locating the Smart Contracts](#locating-the-smart-contracts)
+  - [Starting the SDK](#starting-the-sdk)
+  - [Changing the dApp to work on the private setup](#changing-the-dapp-to-work-on-the-private-setup)
 
 ## Preface
+
 This guide targets users that want to use the Light Client SDK or dApp with a custom network setup. This includes private chain deployments of the Raiden Network or unsupported networks (e.g., Kovan).
 
 To use a private network you first have to deploy the contracts by hand.
 Start by following this tutorial on how to deploy and run Raiden Network on a private chain:
-- [Raiden on Private Network Tutorial] 
 
-After finishing the guide above you should have a running Raiden node along with a private chain with the Raiden Network smart contracts deployed. Now you can move on to setting up the Light Client on your private network.
+- [Raiden on Private Network Tutorial]
+
+After finishing the guide above you should have a running Raiden node along with a private chain with the Raiden Network smart contracts deployed. But before that please go through **Setting up the Light Client on a private network** just below to figure the correct version of `raiden-contracts` and subsequently of `raiden` which will help us deploy raiden-contracts correctly.
```

Comment:
"deploy raiden-contracts" -> "deploy the raiden contracts"

Reply:
done